tests/generate_packageTest.py

```diff
@@ -3,7 +3,8 @@
 import sys
 import unittest
 from unittest.mock import patch
-from bears.generate_package import (VERSION, touch, create_file_from_template,
+from pathlib import Path
```

Comment:
this is part of Python standard library and belongs in the first group of imports, separated by a blank line and then imports from other librarys.

pep 8.

Reply:
alphabetical order pls.

this is all written in pep8.  please study it.